lib/cli/src/helpers.ts

```diff
@@ -29,7 +29,13 @@ export function readFileAsJson(jsonPath: string, allowComments?: boolean) {
 
   const fileContent = fs.readFileSync(filePath, 'utf8');
   const jsonContent = allowComments ? stripJsonComments(fileContent) : fileContent;
-  return JSON.parse(jsonContent);
+
+  try {
+    return JSON.parse(jsonContent);
+  } catch (e) {
+    console.error(chalk.red(`Invalid json on file: ${filePath}`));
```

Comment:
I believe the pattern in this file is to use `logger.error` (logger is just assigned as `const logger = console` above.
```suggestion
    logger.error(chalk.red(`Invalid json on file: ${filePath}`));
```